### PR TITLE
Add hints to list-ops

### PR DESCRIPTION
**Pull request by archanid**
Maybe too wordy for now. We could remove the bullet points and just keep the first blurb.

see exercism/typescript/pull/257

### CODE REVIEW

File: exercises/list-ops/.meta/HINTS.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+This exercise is meant to be implemented with as few built-in functions as possible. This means that a backing array is allowed, but no functions from the `Array.prototype` should be used.
+
+- You don't need any function from `Array.prototype`, and only need the language feature to build construct and desconstruct arrays via destructuring as well as the array literal `[]`.
+- Most implementations using `for` loops also use `.length` on the backing array, or the `array[indexer]`. You don't need either!
```
Comment (tejasbubane) [inline exercises/list-ops/.meta/HINTS.md]: I think we should not talk about `implementations` in exercise README. I get the idea but the wording can be rephrased.

File: exercises/list-ops/.meta/HINTS.md [hunk 2]
```diff
@@ -0,0 +1,5 @@
+This exercise is meant to be implemented with as few built-in functions as possible. This means that a backing array is allowed, but no functions from the `Array.prototype` should be used.
+
+- You don't need any function from `Array.prototype`, and only need the language feature to build construct and desconstruct arrays via destructuring as well as the array literal `[]`.
```
Comment (tejasbubane) [inline exercises/list-ops/.meta/HINTS.md]: > but no functions from the `Array.prototype` should be used.

> but no functions from the `Array.prototype` should be used.

This is repetition - the two sentences can be combined.

Comment (tejasbubane) [reply exercises/list-ops/.meta/HINTS.md]: change `language feature` to `core language features`.

**Comment by archanid** [comment]
@tejasbubane Hey there, updated to match the typescript track one. It should be less of a pain to review.
